### DISCUSSION TITLE
Send print job to multiple printers

### DISCUSSION THREAD

**Discussion by gasprints** [OrcaSlicer/OrcaSlicer#1101] [Ideas]
I have two BL P1P printers and it would be awesome for batch printing if i could send the same slice to both printers at the same time or possibly in a queue where multiple printers could be selected.  It would be handy for the Klipper/OctoPrint users that are set up to push the print to the device and then start it as well.

**Comment by buckeyestargazer** [comment]
Just bumping this up as I'd like to see it too.

**Comment by demosthenes384322** [comment]
Bumping this since I want it to. Most of my prints go to at least two printers at once

**Comment by Aurasphere-au** [comment]
Same

**Comment by 0x0172** [comment]
We have 4 printers at work and they all have to print the same things and the AMS is set up in the same way. It is a pain to manually upload the same print to each printer. I am not looking forward to the future with 10 or more printers ;)

Please add this feature. There are comments on the bambu lab forum and on github requesting this feature since two years.
[Bambu Request](https://forum.bambulab.com/t/bambu-please-add-a-send-to-all-printers-button/3271)
[Github Request](https://github.com/bambulab/BambuStudio/issues/2379)

Unfortunately an update made this procedure even worse. After uploading to one printer there was a cancel button, so the program wont jump to the device page. This button was now removed. I dont want to see the device page, i want to start the next printer. Or even better, the slicer starts all selected machines.

EDIT:
I found the feature just now, thank you very much :)

For all that are still searching:
https://www.youtube.com/watch?v=TSiSmRPilQc

**Comment by Aurasphere-au** [comment]
You could use autohotkey as a workaround in the meantime
